fetch.bs

```diff
@@ -791,7 +792,7 @@ the empty string,
    <td><code>child-src</code>
    <td>HTML's <code>&lt;iframe></code> and <code>&lt;frame></code>
   <tr>
-   <td>""
+   <td>"" or "<code>fetch</code>"
```

Comment:
This whole table is descriptive. It doesn't have any impact on implementers. But saying "or 'fetch'" means you can see that in the wild for such requests, while they would always be the empty string.